### PR TITLE
Handle disabled UndoManager during InlineRename

### PR DESCRIPTION
**Pull request by JoeRobich**
Resolves https://devdiv.visualstudio.com/DevDiv/_workitems/edit/943917

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/InlineRename/InlineRenameUndoManager.cs [hunk 1]
```diff
@@ -123,10 +123,20 @@ public void CreateStartRenameUndoTransaction(Workspace workspace, ITextBuffer su
...
+
+                try
+                {
+                    // Replace the StartRenameSession undo entry with an identically named entry that also includes
+                    // the conflict resolution edits.
+                    undoManager.UndoTo(this.UndoManagers[subjectBuffer].StartRenameSessionUndoPrimitive);
+                }
+                catch (COMException ex) when (ex.ErrorCode == VSConstants.E_UNEXPECTED)
+                {
+                    // According to the documentation, E_UNEXPECTED (0x8000FFFF) is raised when the UndoManager is disabled.
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/InlineRename/InlineRenameUndoManager.cs]: When does this happen, and when it does how bad is the experience?

Comment (JoeRobich) [reply src/VisualStudio/Core/Def/Implementation/InlineRename/InlineRenameUndoManager.cs:128]: Users described VS closing and restarting when this exception was thrown. Not performing this `UndoTo()` did not noticably change the IDE experience even when fully commented out.

**Comment by JoeRobich** [comment]
@jasonmalinowski This is ready to review again.

**Comment by JoeRobich** [comment]
All CI legs are passing but GitHub still shows incomplete legs from when this targeted the master branch.

**Review by ryzngard — APPROVED** [review]

**Pull request merged**